packages/react-dom/package.json

```diff
@@ -1,23 +1,26 @@
 {
   "name": "test-react-dom",
   "private": "true",
-  "version": "16.14.0",
+  "version": "0.0.0",
   "description": "An ESM version of React Dom for Finn",
   "type": "module",
   "scripts": {
     "build": "npx rollup -c",
     "eik:login": "eik login",
     "eik:publish": "eik publish",
     "eik:alias": "eik npm-alias",
-    "postinstall": "npm run build"
+    "postinstall": "npm run build",
+    "eik:publish:ci": "../../scripts/publish.js react-dom @esm-bundle/react-dom"
   },
   "keywords": [],
   "author": "",
   "license": "ISC",
+  "dependencies": {
+    "@esm-bundle/react-dom": "16.14.0"
```

Comment:
while technically a dev dep, it seemed nice to be able to control deps and dev deps separately in renovate. Separate schedules etc.